src/main/java/com/minecolonies/tileentities/TileEntityColonyBuilding.java

```diff
@@ -78,14 +78,12 @@ public void update()
     {
         super.update();
 
+        /*
         if (!worldObj.isRemote && colonyId == 0)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)